src/pygama/hit/build_hit.py

```diff
@@ -259,3 +259,37 @@ def _one_pass(config):
             return new
         else:
             current = new
+
+
+def _get_dependencies(config, par, pars=None):
```

Comment:
add a short docstring to these two funcs

Reply:
@ggmarshall  ?

Reply:
sure I'll see what copilot comes up with

Reply:
maybe `_this_gets_dependencies_i_suppose()`?